EXAMPLES.md

```diff
@@ -149,4 +137,111 @@ Expansion Bay
+
+## Check charger and battery status (Framework 12/13/16)
+
+```
+> sudo framework_tool --power
+Charger Status
+  AC is:            not connected
+  Charger Voltage:  17048mV
+  Charger Current:  0mA
+  Chg Input Current:384mA
+  Battery SoC:      93%
+Battery Status
+  AC is:            not connected
+  Battery is:       connected
+  Battery LFCC:     3693 mAh (Last Full Charge Capacity)
+  Battery Capacity: 3409 mAh
+                    58.96 Wh
+  Charge level:     92%
+  Battery discharging
+```
+
+Get more information
+
+```
+> sudo framework_tool --power -vv
+Charger Status
+  AC is:            not connected
+  Charger Voltage:  14824mV
+  Charger Current:  0mA
+  Chg Input Current:384mA
+  Battery SoC:      33%
+Battery Status
+  AC is:            not connected
+  Battery is:       connected
+  Battery LFCC:     4021 mAh (Last Full Charge Capacity)
+  Battery Capacity: 1300 mAh
+                    19.267 Wh
+  Charge level:     32%
+  Manufacturer:     NVT
+  Model Number:     FRANGWA
+  Serial Number:    038F
+  Battery Type:     LION
+  Present Voltage:  14.821 V
+  Present Rate:     943 mA
+  Design Capacity:  3915 mAh
+                    60.604 Wh
+  Design Voltage:   15.480 V
+  Cycle Count:      64
+  Battery discharging
+```
+
+### Setting a custom charger current limit
+
+```
+# 1C = normal charging rate
+# This means charging from 0 to 100% takes 1 hour
+# Set charging rate to 0.8C
+> sudo framework_tool --charge-rate-limit 0.8
+
+# Limit charge current to the battery to to 2A
+# In the output of `framework_tool --power -vv` above you can se "Design Capacity"
+# Dividing that by 1h gives you the maximum charging current (1C)
+# For example Design Capacity:  3915 mAh => 3915mA
+> sudo framework_tool --charge-current-limit 2000
+
+# And then plug in a power adapter
+> sudo framework_tool --power
+Charger Status
+  AC is:            connected
+  Charger Voltage:  17800mV
+  Charger Current:  2000mA
+                    0.51C
+  Chg Input Current:3084mA
+  Battery SoC:      87%
+Battery Status
+  AC is:            connected
+  Battery is:       connected
+  Battery LFCC:     3713 mAh (Last Full Charge Capacity)
+  Battery Capacity: 3215 mAh
+                    56.953 Wh
+  Charge level:     86%
+  Battery charging
+
+# Remove limit (set rate to 1C)
+> sudo framework_tool --charge-rate-limit 1
+
+# Back to normal
+> sudo framework_tool --power
+Charger Status
+  AC is:            connected
+  Charger Voltage:  17800mV
+  Charger Current:  2740mA
+                    0.70C
+  Chg Input Current:3084mA
+  Battery SoC:      92%
+Battery Status
+  AC is:            connected
+  Battery is:       connected
+  Battery LFCC:     3713 mAh (Last Full Charge Capacity)
+  Battery Capacity: 3387 mAh
+                    60.146 Wh
+  Charge level:     91%
+  Battery charging
+
+# Set charge rate/current limit only if battery is >80% charged
+> sudo framework_tool --charge-rate-limit 80 0.8
+> sudo framework_tool --charge-current-limit 80 2000
```

Comment:
It seems that the limit (out of 1.0) is always required and the state of charge (out of 100) is optional, and in the code the limit is always first and the SoC is second (in an Option), so would that mean that this example has the arguments reversed?  Sorry if I'm missing something obvious; I can't test it because of the issue in the other comment I'm leaving.

Reply:
https://github.com/FrameworkComputer/framework-system/pull/123